Meraki.Api/Data/WirelessRfProfileCreateUpdateRequest.cs

```diff
@@ -11,54 +11,54 @@ public class WirelessRfProfileCreateUpdateRequest
 	/// </summary>
 	[ApiAccess(ApiAccess.ReadWrite)]
 	[DataMember(Name = "name")]
-	public string Name { get; set; } = string.Empty;
+	public string? Name { get; set; }
 
 	/// <summary>
 	/// Steers client to best available access point. Can be either true or false.
 	/// </summary>
 	[ApiAccess(ApiAccess.ReadWrite)]
 	[DataMember(Name = "clientBalancingEnabled")]
-	public bool ClientBalancingEnabled { get; set; }
+	public bool? ClientBalancingEnabled { get; set; }
 
 	/// <summary>
 	/// Gets or Sets MinBitrateType
 	/// </summary>
 	[ApiAccess(ApiAccess.ReadWrite)]
 	[DataMember(Name = "minBitrateType")]
-	public MinBitrateType MinBitrateType { get; set; }
+	public MinBitrateType? MinBitrateType { get; set; }
 
 	/// <summary>
 	/// Gets or Sets BandSelectionType
 	/// </summary>
 	[ApiAccess(ApiAccess.ReadWrite)]
 	[DataMember(Name = "bandSelectionType")]
-	public BandSelectionType BandSelectionType { get; set; }
+	public BandSelectionType? BandSelectionType { get; set; }
```

Comment:
This being nullable allows one to create a Wireless Rf Profile without setting BandSelectionType, which - according to the documentation - is a required parameter during creation.

Reply:
As agreed with @devel0peter, we're going to merge this in but ideally we need 2 separate models.

Reply:
As agreed with @danielabbatt it's OK for now, as the model has been improved. We will move towards separating Creation and Update models, including this one.